src/System.Reflection.Emit.ILGeneration/tests/Generator.cs

```diff
@@ -0,0 +1,427 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Globalization;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Security;
+using System.Text;
+
+namespace TestLibrary
+{
+    public static class Generator
+    {
+        internal static Random m_rand = new Random();
```

Comment:
This use of Random is going to be problematic if this type is used concurrently; the instance can easily get corrupted.